packages/block-library/src/button/edit.native.js

```diff
@@ -134,10 +134,15 @@ function ButtonEdit( props ) {
 	useEffect( () => {
 		// Blur `RichText` on Android when link settings sheet or button settings sheet is opened,
 		// to avoid flashing caret after closing one of them
-		if ( editorSidebarOpened || isLinkSheetVisible ) {
-			if ( Platform.OS === 'android' && richTextRef?.current ) {
-				richTextRef.current.blur();
+		const isAnyBottomSheetVisible =
```

Comment:
minor consideration: `editorSidebarOpened || isLinkSheetVisible` is fairly easy to grok. I'm wondering if we can drop the `isAnyBottomSheetVisible` const for now. Not a block but thought I'd mention it.

Reply:
Sure, it's only used in a single place so we could use the value directly 👍.

Reply:
I applied this suggestion in https://github.com/WordPress/gutenberg/pull/40725/commits/c1c173cbf7ca8ef375079a3294bd2b016773ef0a.